src/Recaptcha.Web-net45/Configuration/RecaptchaConfiguration.cs

```diff
@@ -30,6 +33,7 @@ public RecaptchaConfiguration(string siteKey, string secretKey, string apiVersio
             Theme = theme;
             Size = size;
             UseSsl = useSsl;
+            ApiSource = apiSource;
```

Comment:
This line must be updated because otherwise a null value will be used when the appsetting not present in the configuration.

` ApiSource = string.IsNullOrEmpty(apiSource) ? DEFAULT_API_SOURCE : apiSource;`